test/optional-validation.test.js

```diff
@@ -18,6 +19,8 @@ describe('optional-validation', function() {
 
   before(function(done) {
     db = getSchema();
+    modelWithForceId = db.createModel('modelWithForceId',
```

Comment:
`db.createModel` returns a class constructor, therefore we use names starting with a capital letter - `ModelWithForceId`.